src/auction.mo

```diff
@@ -783,22 +769,11 @@ module {
             Prim.trap("Can never happen: not a single asset processed");
           };
           nextAssetIdToProcess := next;
-          ignore Timer.setTimer<system>(#seconds(1), onTimer);
+          ignore Timer.setTimer<system>(#seconds(1), runAuction);
```

Comment:
Let's remove this Timer as well and make it the responsibility of auction_api.mo to call `runAuction` multiple times. The `Auction` class should not be `<system>`.

Reply:
We should give the outside code control over which auction is run, i.e. pass the `assetId` to `runAuction()`. Then the outside can configure timers itself, or it can run multiple auctions in the same execution if it knows it has enough instructions for that.

Reply:
Done